docs/getting_started/ai-resources.md

```diff
@@ -0,0 +1,73 @@
+# AI Trainings and Resources Guide
+
+## CU Boulder & CURC Resources
+
+### CU Boulder: AI Efforts on Campus
+- Overview of AI initiatives and support across campus. [https://www.colorado.edu/information-technology/ai-cu-boulder/ai-efforts-campus](https://www.colorado.edu/information-technology/ai-cu-boulder/ai-efforts-campus)
```

Comment:
Can we turn these direct links into titles of the pages that are hyperlinks to the pages?

Reply:
Can we do that for all of these on this page?